src/theme/DocPage/index.jsx

```diff
@@ -5,59 +5,12 @@ import { docVersionSearchTag, DocsSidebarProvider, DocsVersionProvider, useDocRo
 import DocPageLayout from "@theme/DocPage/Layout";
 import NotFound from "@theme/NotFound";
 import SearchMetadata from "@theme/SearchMetadata";
-import Head from "@docusaurus/Head";
-import { useLocation } from "@docusaurus/router";
-import useLocaleMap from "../../hooks/useLocaleMap";
-import { usePluginData } from "@docusaurus/useGlobalData";
-import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
-import Cookies from "js-cookie";
+
 function DocPageMetadata(props) {
     const { versionMetadata } = props;
-    const location = useLocation();
-    const { siteConfig } = useDocusaurusContext();
-    const { customFields } = siteConfig;
-    const baseUrl = customFields.baseUrl;
-    const path = location.pathname.replace(baseUrl, "");
-    const internalLocale = path.split("/")[0];
-    const externalLocale = useLocaleMap(internalLocale);
-    const data = usePluginData("docusaurus-plugin-cookiesbanner");
 
-    const cookiesData =
-        data?.cookieData.find((x) => x.languageCode === externalLocale) ||
-        data?.cookieData.find((x) => x.languageCode === siteConfig.customFields.defaultExternalLocales);
-    const items = cookiesData?.items;
```

Comment:
^ Items are no longer loaded from plugin. I believe that dataLayer will be always empty, but I am unable to test that locally (cookiebar and navdata plugin requires Directus).

@msalmeron If you want, we can merge this PR just to see if it works on staging environment. If it does not, then we will simply revert it.

Reply:
I tested it on my local env. with production builds and it's working. The cookies are first loaded from the modal window, so the data in the cookies will always be present, and the loadtag.js script will update the dataLayer with the data on the cookie "cookies-pref".

Hope this clarify how this is working.

Reply:
I was worried that cookies from modal will be loaded later than `loadtag.js` script is executed, so I did basic test. Firstly I used static data for cookie modal:
```patch
diff --git a/src/theme/Modals/CookieModal/index.tsx b/src/theme/Modals/CookieModal/index.tsx
index 83b150c66..982d25a37 100644
--- a/src/theme/Modals/CookieModal/index.tsx
+++ b/src/theme/Modals/CookieModal/index.tsx
@@ -76,10 +76,30 @@ function CookieModal() {
     };
 
     const data = usePluginData("docusaurus-plugin-cookiesbanner") as { cookieData: Array<ICookiesData> | undefined };
-
-    const cookiesData =
-        data?.cookieData.find((x) => x.languageCode === externalLocale) ||
-        data?.cookieData.find((x) => x.languageCode === siteConfig.customFields.defaultExternalLocales);
+    let cookiesData: ICookiesData = {
+        languageCode: "a",
+        manage_body: "b",
+        manage_title: "c",
+        notice_body: "d",
+        notice_title: "e",
+        manage_button_text: "f",
+        accept_all_button_text: "g",
+        confirm_button_text: "h",
+        items: [
+            {
+                required: true,
+                parameter: "foo",
+                title: "bar",
+                description: "baz",
+            },
+            {
+                required: false,
+                parameter: "x",
+                title: "y",
+                description: "z",
+            },
+        ],
+    };
 
     useEffect(() => {
         const prefs = Cookies.get("cookie-prefs");
```

and added some basic logs for tracing:

```patch
diff --git a/src/theme/Modals/CookieModal/index.tsx b/src/theme/Modals/CookieModal/index.tsx
index 83b150c66..982d25a37 100644
--- a/src/theme/Modals/CookieModal/index.tsx
+++ b/src/theme/Modals/CookieModal/index.tsx
@@ -116,6 +136,7 @@ function CookieModal() {
         const values = selected.map((x) => {
             return { name: x.name, value: x.value };
         });
+        console.trace("[CookieModal] Setting cookie-prefs");
         Cookies.set("cookie-prefs", JSON.stringify(values));
         window.location.reload();
     };
@@ -124,6 +145,7 @@ function CookieModal() {
         const values = cookiesData?.items.map((x) => {
             return { name: x!.parameter!, value: true };
         });
+        console.trace("[CookieModal] Setting cookie-prefs (all)");
         Cookies.set("cookie-prefs", JSON.stringify(values), { expires: 365 });
         window.location.reload();
     };
diff --git a/static/js/loadtags.js b/static/js/loadtags.js
index 8bd7aeb53..b147e74ea 100644
--- a/static/js/loadtags.js
+++ b/static/js/loadtags.js
@@ -6,6 +6,8 @@ if (cookiePrefs) {
     cookies = JSON.parse(decodeURIComponent(cookiePrefs.split("=")[1]));
 }
 
+console.trace("[loadtags.js] Cookies for dataLayer:", cookies);
+
 if (!window.gtmConfigured && cookies) {
     const layer = "dataLayer";
     window[layer] = window[layer] || [];
@@ -23,4 +25,6 @@ if (!window.gtmConfigured && cookies) {
     });
 
     window.gtmConfigured = true;
+
+    console.trace("[loadtags.js] GTM configured");
 }
```

First page load:
![image](https://github.com/casper-network/docs-new/assets/121791569/abb9dd3f-3af7-4932-a843-fbe90f995277)

After confirming selection page got reloaded:

![image](https://github.com/casper-network/docs-new/assets/121791569/cb912b0d-631f-4949-99bb-0c817b21c3ab)

Indeed, it works :+1: I see that `CookieModal` is calling `window.location.reload()` after setting cookies.